config/packages/messenger.yaml

```diff
@@ -13,12 +13,10 @@ framework:
                 middleware:
                     - handle_message
 
+            messenger.bus.event: ~
```

Comment:
I this having the config like that will be difficult for newer people in messenger component to understand it. Can we define explicitly the transport we want here and the routing.

I this this is configuration over convention but IMO will be easier to understand for more people and will not create a higher curve when starting with the project

Reply:
As far as I know there is no such possibility. I searched that when i start doing this PR because i have same opinion like you.
Transports are related to routings. Message bus is something else.

Reply:
One last thing. Broadway has his own event bus that is synchronous. Can we rename this to something like messenger.bus.event.async or something like that the indicates that the handlers will be executed in a different process?

Reply:
good point, updated.